src/Datasource/EntityTrait.php

```diff
@@ -418,6 +418,36 @@ public function hiddenProperties($properties = null)
     }
 
     /**
+     * Sets hidden properties.
+     *
+     * @param null|array $properties Either an array of properties to treat as virtual or null to get properties
+     * @param bool $merge Merge the new properties with the existing. By default false.
+     * @return self
+     */
+    public function setHiddenProperties(array $properties, $merge = false)
```

Comment:
Can we drop the `Properties` suffix from all new methods?  It makes the names too long, plus it's not consistently used for all methods so doesn't bring much value.